styles/_buttons.scss

```diff
@@ -1,88 +1,69 @@
 /* stylelint-disable */
 /***** Buttons *****/
 .button, [role='button'] {
-  @include tablet { width: auto; }
-  display: inline-block;
-  width: 100%;
-  margin: 0;
-  padding: 0 20px;
-  transition: background-color .12s ease-in-out, border-color .12s ease-in-out, color .15s ease-in-out;
-  border: 1px solid $button-color;
-  border-radius: 4px;
-  box-shadow: unset;
-  color: $button-color;
-  font-size: 12px;
-  line-height: 2.34; //Taken from Zendesk garden bedrock
+  display: inline-flex;
+	position: relative;
+	align-items: center;
+	padding: 0.75rem 1.75rem;
+  overflow: hidden;
+  transition: background $transition-speed, color $transition-speed, fill $transition-speed, padding $transition-speed;
```

Comment:
might want to remove the stylelint disable on buttons. There are only a few errors. Most of which are new code